CourseGrab/Home/InternetErrorView.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  InternetErrorView.swift
+//  CourseGrab
+//
+//  Created by Reade Plunkett on 3/1/20.
+//  Copyright © 2020 Cornell AppDev. All rights reserved.
+//
```

Comment:
This view and `NoTrackedCoursesView` look extremely similar except for some small things like text. can we refactor this into one view that just has either takes in some arguments in its initializer or has some `configure` function